src/manufacturer/manufacturer.component.tsx

```diff
@@ -63,16 +63,16 @@ function ManufacturerComponent() {
             </MuiLink>
           ),
       },
-
       {
         header: 'Last modified',
-        accessorFn: (row) => row.modified_time,
+        accessorFn: (row) => new Date(row.modified_time),
         id: 'modified_time',
-        size: 250,
+        filterVariant: 'datetime-range',
+        size: 350,
         enableGrouping: false,
         Cell: ({ row }) =>
           row.original.modified_time &&
-          formatDateTimeStrings(row.original.modified_time),
+          formatDateTimeStrings(row.original.modified_time, false),
```

Comment:
```suggestion
          formatDateTimeStrings(row.original.modified_time, true),
```

Reply:
@MatteoGuarnaccia5 I think you may have missed this one